Bruin Bite/Network/UserAPI.swift

```diff
@@ -29,13 +29,13 @@ extension API {
                     do {
                         let errresults = try JSONDecoder().decode(UserError.self, from: response.data)
                         MAIN_USER.initUserError(error: errresults)
-                        print(err)
+                        Logger.shared.handle(type: .error, message: "createUser: \(err)")
                     } catch let reserr {
-                        print(reserr)
+                        Logger.shared.handle(type: .error, message: "createUser: \(reserr)")
```

Comment:
Same for all these messages. Change from format "\<functionName\>: \(error)" to format "\<functionName\> failed with error: \(error)".